src/components/Filter.res

```diff
@@ -288,12 +288,82 @@ let make = (
     setfilterKeys(_ => keys)
   }
 
+  let allFiltersUI =
```

Comment:
Open LogicUtils on Top as it's been used multiple places

Reply:
okay